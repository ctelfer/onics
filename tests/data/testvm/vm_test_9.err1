Packet     1: VM provided no return value
Packet     2: VM provided no return value
Packet     3: VM provided no return value
Packet     4: VM provided no return value
Packet     5: VM provided no return value
Packet     6: VM provided no return value
Packet     7: VM provided no return value
Packet     8: VM provided no return value
Packet     9: VM provided no return value
Packet    10: VM provided no return value
Packet    11: VM provided no return value
Packet    12: VM provided no return value
Packet    13: VM provided no return value
Packet    14: VM provided no return value
Packet    15: VM provided no return value
5353482d322e302d4f70656e5353485f
Packet    16: VM provided no return value
Packet    17: VM provided no return value
Packet    18: VM provided no return value
Packet    19: VM provided no return value
Packet    20: VM provided no return value
Packet    21: VM provided no return value
Packet    22: VM provided no return value
Packet    23: VM provided no return value
Packet    24: VM provided no return value
Packet    25: VM provided no return value
Packet    26: VM provided no return value
Packet    27: VM provided no return value
Packet    28: VM provided no return value
Packet    29: VM provided no return value
Packet    30: VM provided no return value
Packet    31: VM provided no return value
Packet    32: VM provided no return value
Packet    33: VM provided no return value
Packet    34: VM provided no return value
Packet    35: VM provided no return value
Packet    36: VM provided no return value
Packet    37: VM provided no return value
Packet    38: VM provided no return value
Packet    39: VM provided no return value
Packet    40: VM provided no return value
Packet    41: VM provided no return value
Packet    42: VM provided no return value
Packet    43: VM provided no return value
Packet    44: VM provided no return value
Packet    45: VM provided no return value
Packet    46: VM provided no return value
Packet    47: VM provided no return value
Packet    48: VM provided no return value
Packet    49: VM provided no return value
Packet    50: VM provided no return value
Packet    51: VM provided no return value
Packet    52: VM provided no return value
Packet    53: VM provided no return value
Packet    54: VM provided no return value
Packet    55: VM provided no return value
Packet    56: VM provided no return value
Packet    57: VM provided no return value
Packet    58: VM provided no return value
Packet    59: VM provided no return value
Packet    60: VM provided no return value
Packet    61: VM provided no return value
Packet    62: VM provided no return value
Packet    63: VM provided no return value
Packet    64: VM provided no return value
Packet    65: VM provided no return value
Packet    66: VM provided no return value
Packet    67: VM provided no return value
Packet    68: VM provided no return value
Packet    69: VM provided no return value
Packet    70: VM provided no return value
Packet    71: VM provided no return value
Packet    72: VM provided no return value
Packet    73: VM provided no return value
Packet    74: VM provided no return value
Packet    75: VM provided no return value
Packet    76: VM provided no return value
Packet    77: VM provided no return value
Packet    78: VM provided no return value
Packet    79: VM provided no return value
Packet    80: VM provided no return value
Packet    81: VM provided no return value
Packet    82: VM provided no return value
Packet    83: VM provided no return value
Packet    84: VM provided no return value
Packet    85: VM provided no return value
Packet    86: VM provided no return value
Packet    87: VM provided no return value
Packet    88: VM provided no return value
Packet    89: VM provided no return value
Packet    90: VM provided no return value
0 out of 90 packets returned 'true'
