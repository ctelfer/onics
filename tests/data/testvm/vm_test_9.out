#####
# Packet 1 -- 106 bytes
#####
#####
# ETH: Ethernet II -- Offset 192, Length 106
#####
# ETH: Destination Address: 00:22:fa:a7:69:90
# ETH: Source Address:      00:30:65:c1:b3:a8
# ETH: Ethernet Type:       0800
    000000:  00 22 fa a7 69 90 00 30 65 c1 b3 a8 08 00        |."..i..0e.....|
#####
# IP: Internet Protocol Version 4 -- Offset 206, Length 92
#####
# IP: Version:              4
# IP: Header Length:        5 (20 bytes)
# IP: Diffserv:             00
# IP: ECN:                  0
# IP: Total Length:         92 bytes
# IP: Identifier:           2fdc
# IP: Reserved Frag Bit:    0..
# IP: Don't Fragment Bit:   .1.
# IP: More Fragments Bit:   ..0
# IP: Fragment Offset:      0 quadwords
# IP: Time to Live:         64
# IP: IP Protocol:          6
# IP: Header Checksum:      8964
# IP: Source Address:       192.168.0.7
# IP: Destination Address:  192.168.0.4
    00000e:  45 00 00 5c 2f dc 40 00 40 06 89 64 c0 a8 00 07  |E..\/.@.@..d....|
    00001e:  c0 a8 00 04                                      |....|
#####
# TCP: Transmission Control Protocol -- Offset 226, Length 72
#    Errors [0x8]: checksum
#####
# TCP: Source Port:          22
# TCP: Destination Port:     38859
# TCP: Sequence Number:      770160473
# TCP: Acknowlege Number:    1962529759
# TCP: Data Offset:          8 (32 bytes)
# TCP: Reserved Bits:        0
# TCP: Congest Win Reduced:  0.......
# TCP: ECN Enabled:          .0......
# TCP: Urgent:               ..0.....
# TCP: Acknowledgement:      ...1....
# TCP: Push:                 ....1...
# TCP: Reset:                .....0..
# TCP: Synchronize:          ......0.
# TCP: Finalize:             .......0
# TCP: Window:               362
# TCP: Checksum:             d68e
# TCP: Urgent Pointer:       0
    000022:  00 16 97 cb 2d e7 b7 59 74 f9 d3 df 80 18 01 6a  |....-..Yt......j|
    000032:  d6 8e 00 00 01 01 08 0a 03 59 99 05 03 4d d4 71  |.........Y...M.q|
# TCP:  Data -- 40 bytes [66, 106]
    000042:  48 65 6c 6c 6f 20 57 6f 72 6c 64 0a 53 53 48 5f  |Hello World.SSH_|
    000052:  34 2e 37 70 31 20 44 65 62 69 61 6e 2d 38 75 62  |4.7p1 Debian-8ub|
    000062:  75 6e 74 75 31 2e 32 0a                          |untu1.2.|


